app/views/static/index.html.erb

```diff
@@ -48,29 +48,40 @@
   <div class="columns">
     <% @events.each do |event| %>
       <div class="column is-4">
-        <div class="card vertical-scroll" style="height: 300px;">
-          <div class="card-content">
-            <p><a class="title is-4" href="<%= event_path(event) %>"><%= event.name %></a></p>
+        <div class="card event-card">
+          <!-- <a> nested inside .eventCard makes entire card clickable, but messes with text selection -->
+          <a href="<%= url_for(event) %>"></a>
```

Comment:
I also changed the url not to directly go to the "more details" url but instead the event details page (which I actually still need to make pretty)